### DISCUSSION TITLE
V5 折线图数据为 0 时，未对齐 y 轴 0 刻度

### DISCUSSION THREAD

**Discussion by pfdgithub** [antvis/G2#5737] [Q&A]
### 问题描述

当前版本 `@antv/g2@5.1.6`
印象中之前有个版本是没问题的，但回退尝试了所有可能的版本，都没有解决。

```
    chart.options({
      data: [],
      type: "line",
      transform: [{ type: "sortColor" }],
      encode: {
        x: "datetime",
        y: "requestCount",
        color: "serviceName",
      },
      scale: {
        y: {
          nice: true,
          zero: true,
        },
      },
    });
```
![image](https://github.com/antvis/G2/assets/3262762/472425eb-01da-4589-aabd-d7f7e1080f7b)

### 平台

- 操作系统: macOS
- 网页浏览器: Google Chrome

**Comment by pearmini** [comment]
给一个完整的复现 demo，比如一个 code sandbox

**Reply by yikai89827** [reply]
const chart = new Chart({
            container: 'case-report-trend',
            autoFit: true,
            marginBottom: 0,
            marginLeft: 0,
            marginRight: 10,
            title: '',
        });
        chart.data([
            { date: '06-10', count: 0, type: '测试' },
            { date: '06-11', count: 0, type: '测试' },
            { date: '06-12', count: 0, type: '测试' },
            { date: '06-13', count: 0, type: '测试' },
            { date: '06-14', count: 0, type: '测试' },
            { date: '06-15', count: 0, type: '测试' },
            { date: '06-16', count: 0, type: '测试' }
        ]);

        chart
            .line()
            .encode('x', 'date')
            .encode(
                'y',
                localUrl?.pathname === '/user' ? 'activeCount' : 'count',
            )
            .encode('shape', 'smooth')
            .encode('color', 'type')
            .scale('color', {
                domain: ['测试'],
                range: ['#FF9410'],
            })
            .style('lineWidth', 2)
            .legend(false)
            .scale('y', { independent: true, nice: true })
            .axis('x', {
                labelFill: '#fff',
                tickStroke: '#fff',
                lineStroke: '#fff',
                labelStroke: '#fff',
                labelFontSize: 12,
                tick: true,
                line: true,
                title: '',
            })
            .axis('y', {
                labelFill: '#fff',
                tickStroke: '#fff',
                lineStroke: '#fff',
                labelStroke: '#fff',
                gridStroke: '#fff',
                labelFontSize: 12,
                gridLineWidth: 1,
                gridLineDash: [1, 0],
                tick: true,
                grid: true,
                title: '',
                labelFormatter: (v: any) =>
                    v >= 10000 ? (v / 10000).toFixed(0) + '万' : v,
            })
            .tooltip({ items: [{ channel: 'y', name: '测试' }] });
        chart.render();

这是复现的demo

**Reply by yikai89827** [reply]
![image](https://github.com/antvis/G2/assets/55533597/ddc20f36-bf98-442b-8547-cad4d77671d7)

**Reply by yikai89827** [reply]
数据都是0的时候  刻度会垂直居中 没有和X轴重叠

**Comment by pfdgithub** [comment]
目前无法稳定复现，等找到复现方式再提供
